.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+# Mac
+.DS_Store
```

Comment:
this change to gitignore should not go in this PR since this PR is about adding reverse-string, not gitignore

Reply:
I'm not sure why this was added. Hopefully the last commit I did corrects this.